doc/customizing_guide/mpl_viewer/config.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, axes, *args, **kwargs):
         self.state.add_callback('zorder', self._on_visual_change)
         self.state.add_callback('color', self._on_visual_change)
         self.state.add_callback('alpha', self._on_visual_change)
+        self.state.add_callback('preferred_cmap', self._on_visual_change)
```

Comment:
I don't think we should listen for changes in preferred_cmap - the way I see it it should just be a parameter that affects future image viewers that are open, not change existing ones if preferred_cmap is updated.

Reply:
Now I see what I should do... Thanks @astrofrog!